.github/workflows/make-demo-repo.yml

```diff
@@ -13,15 +13,14 @@ jobs:
         shell: bash -l {0}
     steps:
     - name: Checkout Repo
-      uses: actions/checkout@v2
+      uses: actions/checkout@v3
       with:
         path: ./repo
         fetch-depth: 0  # avoids shallow checkout as needed by setuptools-scm
     - name: Set up Python
-      uses: actions/setup-python@v1
+      uses: actions/setup-python@v4
       with:
-        python-version: 3.7
-        auto-activate-base: false
```

Comment:
I tried to check the docs of `setup-python` but I could not find a `auto-activate-base` configuration, so I decided to remove it...